src/RadworksGovernor.sol

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {Governor} from "@openzeppelin/contracts/governance/Governor.sol";
+import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {
+  ERC20VotesComp,
+  GovernorVotesComp
+} from "@openzeppelin/contracts/governance/extensions/GovernorVotesComp.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {ICompoundTimelock} from
+  "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {GovernorSettings} from "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
+
+import {IRadworksTimelock} from "src/interfaces/IRadworksTimelock.sol";
+import {RadworksGovernorTimelockCompound} from "src/lib/RadworksGovernorTimelockCompound.sol";
+import {GovernorCompatibilityBravo} from
+  "@openzeppelin/contracts/governance/compatibility/GovernorCompatibilityBravo.sol";
+
+/// @notice The upgraded Radworks Governor: Bravo compatible and extended from OpenZeppelin.
+contract RadworksGovernor is
+  Governor,
+  GovernorVotesComp,
+  RadworksGovernorTimelockCompound,
+  GovernorSettings
+{
+  struct ProposalVote {
+    uint256 againstVotes;
+    uint256 forVotes;
+    uint256 abstainVotes;
+  }
+
+  /// @notice The address of the RAD token on Ethereum mainnet from which this Governor derives
+  /// delegated voting weight.
+  ERC20VotesComp private constant RAD_TOKEN =
+    ERC20VotesComp(0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3);
+
+  /// @notice The address of the existing Radworks DAO Timelock on Ethereum mainnet through
+  /// which this Governor executes transactions.
+  IRadworksTimelock private constant TIMELOCK =
+    IRadworksTimelock(payable(0x8dA8f82d2BbDd896822de723F55D6EdF416130ba));
+
+  /// @notice Human readable name of this Governor.
+  string private constant GOVERNOR_NAME = "Radworks Governor Bravo";
+
+  /// @notice The number of RAD (in "wei") that must participate in a vote for it to meet quorum
+  /// threshold.
+  ///
+  /// TODO: placeholder
+  uint256 private constant QUORUM = 100_000e18; // 100,000 RAD
+
+  /// @param _initialVotingDelay The initial voting delay this Governor will enforce.
+  /// @param _initialVotingPeriod The initial voting period this Governor will enforce.
+  /// @param _initialProposalThreshold The initial number of RAD required to submit
+  /// a proposal this Governor will enforce.
+  constructor(
+    uint256 _initialVotingDelay,
+    uint256 _initialVotingPeriod,
+    uint256 _initialProposalThreshold
+  )
+    GovernorVotesComp(RAD_TOKEN)
+    GovernorSettings(_initialVotingDelay, _initialVotingPeriod, _initialProposalThreshold)
+    RadworksGovernorTimelockCompound(TIMELOCK)
+    Governor(GOVERNOR_NAME)
+  {}
+
+  ///
```

Comment:
nit: think it's more common when using `///` to omit any empty natspec lines. so this would be come just the line on 68. Applies to multiple places in this file

```suggestion
```

Reply:
Done 66f5ea5567306bdc247268d517a2619242bcee32